commandline/site.yml

```diff
@@ -49,6 +50,7 @@
 #   - { role: pytorch, tags: pytorch }
    - { role: bert_models , tags: bert_models }
    - { role: tsv-utils , tags: tsv-utils }
+- { import_playbook: heliots/heliots.yml , tags: heliots }
```

Comment:
@kaisakui @aajarven  I removed the "role" and I was unsure myself how to use import_playbook, I experimented a bit, it seems this is the way to do it. Very similar.